sauron/sauron.py

```diff
@@ -11,7 +11,7 @@
 
 plugin = Plugin(dynamic=False)
 plugin.sauron_socks_proxies = None
-plugin.sauron_network = "test"
+plugin.sauron_network = "bitcoin"
```

Comment:
Why this change?

Seems like not a great idea to make mainnet the default?

Reply:
Upss, this was unintentional. A leftover from testing.
Fixed.